### DISCUSSION TITLE
Any support for dotnet MAUI?

### DISCUSSION THREAD

**Discussion by nabond251** [AsynkronIT/protoactor-dotnet#2079] [Q&A]
I've asked a similar question on the Orleans repo. Currently looks like Akka is the only actor model implementation that has any support for MAUI apps. Any plans to try that here?

**Comment by bencz** [comment]
@nabond251 The protoactor works normally with .net maui, see the example....
I just created an actor that, every second, sends a message to the UI actor saying there was a 'click'.

[ProtoActorExample.zip](https://github.com/user-attachments/files/17366930/ProtoActorExample.zip)

**Comment by bencz** [comment]
Also, it works on android ( I don't have an iOS to test it )

![ezgif-3-fffab06ce0](https://github.com/user-attachments/assets/59ac9aa0-7e70-4fbb-9840-a55d0a178256)
